Dockerfile

```diff
@@ -14,7 +14,8 @@ RUN add-apt-repository -y ppa:webupd8team/java \
 && echo oracle-java8-installer shared/accepted-oracle-license-v1-1 select true | /usr/bin/debconf-set-selections \
 && apt-get install -y \
 software-properties-common \
-oracle-java8-installer
+oracle-java8-installer \
+oracle-java8-set-default
```

Comment:
I'm surprised that the Oracle JDK is giving us this problem (OpenJDK maybe, but not Oracle). 

Would moving to one of these docker images help at all? https://hub.docker.com/r/azul/zulu-openjdk/